### DISCUSSION TITLE
[JS101] 關於 for 迴圈的參數

### DISCUSSION THREAD

**Discussion by rich2020s** [Lidemy/forum#78] [General]
正常 for 的迴圈是像這樣：
`for (let i = 0; i < 10; i++){...}`
  
我也忘了我是在哪裡學到的，可以放更多變量：
`for (let i = 0, j = 10; i < 10; i++,j--){...}`

這樣寫在JS上還是可以跑的，但是聽助教說，才知道好像只有 C 這樣寫迴圈。
我的問題是，for 迴圈是可以這樣寫的嗎？會不會遇到什麼不可預測的問題？
未來在工作上也可以這樣寫嗎？
謝謝大家！

**Comment by aszx87410** [comment]
是可以這樣寫的，本來就沒有限制只能放一個變數喔
也不是說只有 C 這樣寫，只是這樣的寫法相對來說比較少見一點

**Reply by rich2020s** [reply]
好的，謝謝老師！
